compiler/noirc_driver/src/lib.rs

```diff
@@ -275,13 +275,13 @@ pub fn compile_contract(
 
     if contracts.len() > 1 {
         let err = CustomDiagnostic::from_message("Packages are limited to a single contract")
-            .in_file(FileId::default());
+            .in_file(SrcId::default());
         return Err(vec![err]);
     } else if contracts.is_empty() {
         let err = CustomDiagnostic::from_message(
             "cannot compile crate into a contract as it does not contain any contracts",
         )
-        .in_file(FileId::default());
+        .in_file(SrcId::default());
```

Comment:
This rename looks like it can be in a separate PR

Reply:
We don't really have to change this name either - it just seemed more logical to name it SrcId that is more universal rather than FileId which is fs specific, though data stays the same.